cmake/templates/OpenVINOConfig.cmake.in

```diff
@@ -342,6 +342,7 @@ endmacro()
 
 macro(_ov_find_intel_cpu_dependencies)
     set(_OV_ENABLE_CPU_ACL "@DNNL_USE_ACL@")
+    set(_OV_ENABLE_CPU_EXTENSIONS "@ENABLE_CPU_EXTENSIONS@")
```

Comment:
maybe we need to define `_OV_COMPILE_CPU_EXTENSIONS ` instead ?
In this case, we can simplify condition:
```
if (NOT _OV_COMPILE_CPU_EXTENSIONS)
   _ov_find_dependency(cpu_extensions)
endif()
```
without any checks for compiler versions.

Reply:
Good idea, thanks. Updated.